src/common/options.cc

```diff
@@ -8088,6 +8088,10 @@ std::vector<Option> get_mds_options() {
     .set_default(5)
     .set_description("maximum number of scrub operations performed in parallel"),
 
+    Option("mds_only_process_op_on_auth_mds", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
```

Comment:
mds_forward_all_requests_to_auth

Reply:
changed

Reply:
I think set_trace_dist is a frequently called function.  so I define a variable in Server.